src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/draft/DraftOffenceController.kt

```diff
@@ -43,7 +43,7 @@ class DraftOffenceController(
 
   @PutMapping(value = ["/{id}/offence-details"])
   @Operation(summary = "Set the offence details for the draft adjudication.", description = "At least one set of offence details must be supplied")
-  @PreAuthorize("hasAuthority('SCOPE_write')")
+  @PreAuthorize("hasRole('ADJUDICATIONS_REVIEWER') and hasAuthority('SCOPE_write')")
```

Comment:
this is VIEW too

Reply:
indeed - see https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467#discussion_r1286774976

Reply:
indeed - see https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/467#discussion_r1286774976